app/bin/service/search.dart

```diff
@@ -78,7 +78,7 @@ void _main(FrontendEntryMessage message) {
     final scheduler = new TaskScheduler(
       batchIndexUpdater,
       [
-        new ManualTriggerTaskSource(taskReceivePort),
+        new ManualTriggerTaskSource(taskReceivePort as Stream<Task>),
```

Comment:
Are you sure this works in Dart 2 mode? If I'm reading things right, a `ReceivePort` is a `Stream<dynamic>`, so this cast should fail at runtime.